cyral/internal/datalabel/model.go

```diff
@@ -28,7 +28,11 @@ type DataLabel struct {
 	Implicit           bool                   `json:"implicit,omitempty"`
 }
 
-func (dl *DataLabel) WriteToSchema(d *schema.ResourceData) error {
+func (dl DataLabel) WriteToSchema(d *schema.ResourceData) error {
+	// if err := d.Set("name", dl.Name); err != nil {
+	// 	return fmt.Errorf("error setting 'name' field: %w", err)
+	// }
```

Comment:
Why this is commented?

Reply:
Code used for testing purposes that should have been removed. Thanks for catching it. Fixed in [34c2a8d](https://github.com/cyralinc/terraform-provider-cyral/pull/474/commits/34c2a8d948c132bad8b2d953865472c57ce671f1).